ge/urls.py

```diff
@@ -6,6 +6,7 @@
     path("ge/report/", views.report),
     path("ge/search/", views.search),
     path("ge/edit/<int:item_id>", views.edit_librarydata, name="edit_librarydata"),
+    path("ge/add_fund/", views.add_fund, name="add_fund"),
```

Comment:
@ztucker4 Now that we're doing more than one thing, let's be consistent.  For the views, `edit_librarydata` vs `add_fund` - I suppose `fund` is a better term overall, so please refactor `edit_librarydata` to `edit_fund`.

Similarly, the URLs should be consistent: `edit` vs `add_fund`.  We only act on one thing, but the more-specific `edit_fund` is better than the generic `edit`, so please incorporate that into refactoring too.